packages/nuxt3/src/server/api/utils/text-formatting.ts

```
@@ -0,0 +1,49 @@
+
+/**
+ * Format ping response as text
+ */
+export function formatPingResponseAsText(pingResponse: PingResponse): string {
+  return `${pingResponse.message} - ${pingResponse.service} - ${pingResponse.timestamp}`;
+}
+
+/**
+ * Format any object as pretty JSON string
+ */
+export function formatAsJsonString(data: unknown, indentation: number = 2): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)